token/cli/src/main.rs

```diff
@@ -807,6 +852,50 @@ fn main() {
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name("create-multisig")
+                .about("Create a new account describing an M:N multisignature")
+                .arg(
+                    Arg::with_name("minimum_signers")
+                        .value_name("MINIMUM_SIGNERS")
+                        .validator(is_multisig_minimum_signers)
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help(&format!("The minimum number of signers required \
+                            to allow the operation. [{} <= M <= N]",
+                            MIN_SIGNERS,
+                        )),
+                )
+                .arg(
+                    Arg::with_name("signatory")
+                        .value_name("SIGNATORY_PUBKEY")
```

Comment:
I wanted to avoid `signer` here as that conflicts with the offline arg for specifying a presigner pubkey:sig pairing.  I ended up going with `multisig_signer` when I did the plumbing, could reuse that.

Reply:
Sure, `multisig_signer` works

Reply:
I went with `multisig_member` to prevent added complexity to keep `create-multisig` separate from the other subcommands